dev-tools/packaging/packages.yml

```
@@ -138,6 +147,14 @@ shared:
     files:
       <<: *binary_files
 
+  - &fips_binary_spec
```

```suggestion
  - &binary_spec_fips
```

for consistency let's stick with `_fips` as suffix here

---

done in c6e70d9fdfa2992199520f2fcffa39c757b3c8ad